tests/test_flash_attn.py

```diff
@@ -1181,6 +1182,14 @@ def test_flash_attn_output(
 def test_flash_attn_varlen_output(
     seqlen_q, seqlen_k, d, dropout_p, causal, local, alibi, deterministic, mha_type, dtype, kvpacked, forward_only=True
 ):
+    if is_hip():
+        if dropout_p != 0.0:
+            pytest.skip("Dropout not supported in HIP")
+        
+        # skip all cases where seqlen_q, seqlen_k, or d are not powers of 2
+        if not (is_power_of_2(seqlen_q) and is_power_of_2(seqlen_k) and is_power_of_2(d)):
```

Comment:
Are these tests failing?

Reply:
Most of them work, but there are some cases that fail. Also the number of tests is a lot. Just running the tests selected in this subset take like an hour